Disaster-Response-Platform/backend/Controllers/need_controller.py

```diff
@@ -121,24 +122,4 @@ def get_unsupplied_quantity(need_id: str, response: Response):
         response.status_code = HTTPStatus.NOT_FOUND
         return json.loads(err_json)
     
-@router.put("/{need_id}/urgency")
```

Comment:
Seems these two api functions are accidentally deleted from needs CRUD.

Reply:
yes, indeed, thanks for noticing it, I'll change it